EXT_MD.md

```diff
@@ -0,0 +1,132 @@
+records and further attributes.
+
+The example data structure can be found [HERE](https://github.com/ejcjason/MDDomainExtension).
+
+Root Group
+---------------------------
+
+### Additional Attributes for the `Root` group
+
+The following additional attributes are defined in this extension.
+The individual requirement is given in `scope`.
+
+  - `forceField`
+    - type: 1-dimensional array of N (string) elements, where N is the number of force fields (interatomic potentials) implemented in the simulation.
+    - scope: *optional*
+    - description: the methods implemented in the simulation to describe the force fields (interatomic potentials). See [Interatomic Potentials Repository](https://www.ctcms.nist.gov/potentials/).
+    - example values:
+      - `eam/alloy`
+      - `lj/cut 3.0`
+      - ...
+  - `forceFieldParameters`
+    - type: 1-dimensional array of N (string) elements, where N is the number of force fields (interatomic potentials) implemented in the simulation.
+    - scope: *optional*
+    - description: the parameters specification for the `forceField` methods. See [Interatomic Potentials Repository](https://www.ctcms.nist.gov/potentials/).
+    - example values:
+      - `pair_coeff * * 1 1`
+      - `pair_coeff 1 1 Cu_mishin1.eam.alloy Cu`
+      - ...
+
+Observable Records
+---------------------------
+
+`observables` is an *optional* group that contains physical observables that are derived from the system state, i.e., thermodynamic information (e.g. temperature, energy, pressure).
+
+### Naming conventions
+
+The naming conventions conforming the naming conventions for [Scalar, Vector and Tensor Records](https://github.com/openPMD/openPMD-standard/blob/latest/STANDARD.md#naming-conventions) in [openPMD base standard](https://github.com/openPMD/openPMD-standard/blob/latest/STANDARD.md).
+
+- `scalar` record
+  - type: *(any type)*
+  - data set: `recordName` unique name in group `basePath` + `observables`
+  - examples:
+    - /data/observables/temperature
+    - /data/observables/pressure
+
+### Attributes for each `observable` record
+
+The attributes of unit system for records should be included as defined in [openPMD base standard](https://github.com/openPMD/openPMD-standard/blob/latest/STANDARD.md#unit-systems-and-dimensionality).
+
+
+Particle Records
+---------------------------
+
+### Additional attributes for each `position` record
+
+The following attributes are defined in this extension. The individual requirement is given in `scope`. 
+
+- `coordinate`
+    - type: *(string)*
+    - scope: *required*
+    - description: the representation of the stored position coordinates
+    - available values:
+      - `absolute` the unscaled coordinates
+      - `fractional` the coordinates that are scaled in the range of [0,1] relative to the length of each box edge; in this coordinate system, the `unitSI` of each position component should be `1.0`
```

Comment:
For fractional, do you intentionally include the upper value? (E.g. `[0.0, 1.0]` vs. `[0.0, 1.0)`) Just double-checking.
Formatting suggestion:
```suggestion
      - `fractional` the coordinates that are scaled in the range of `[0.0, 1.0]` relative to the length of each box edge; in this coordinate system, the `unitSI` of each position component should be `1.0`
```

Reply:
I didn't think it deeply. But intuitively, for the atom located at the edge corner of the simulation box, it's fractional coordinate is [1.0,1.0,1.0]. Thus I set the range to be [0.0,1.0].

Please correct me if there are some occasions I missed.

Reply:
Ok, I am just wondering about double-counting @JunCEEE.